datafu-spark/README.md

```diff
@@ -0,0 +1,71 @@
+# datafu-spark
+
+datafu-spark contains a number of spark API's and a "Scala-Python bridge" that makes calling Scala code from Python, and vice-versa, easier.
+
+It has been tested on Spark releases from with 2.1.0 to 2.4.0, using Scala 2.10 and 2.11.
+
+-----------
```

Comment:
It's be nice if the top of the README had some short examples showing what you can do with the library purely in scala.

Also can we show an example of calling Python code from Scala since this is mentioned as one of its features?

Reply:
Yes, I still don't know what spark-datafu is.

Reply:
I have a work-in-progress blog post explaining some of the datafu-spark API's. I can add it to the branch, and maybe move some of it into the datafu-spark README.